nncf/data/dataset.py

```diff
@@ -72,6 +72,15 @@ def get_inference_data(self, indices: Optional[List[int]] = None) -> Iterable[Mo
         """
         return DataProvider(self._data_source, self._transform_func, indices)
 
+    def get_length_or_none(self) -> Optional[int]:
```

Comment:
Just call it `get_length`, the type hint with a docstring should be enough

Reply:
Done